app/controllers/administrate/application_controller.rb

```diff
@@ -104,6 +107,8 @@ def dashboard
 
     def requested_resource
       @_requested_resource ||= find_resource(params[:id])
+      authorize_resource(@_requested_resource)
+      @_requested_resource
     end
```

Comment:
This breaks the existing memoization. Here's an alternative, saving us from (potentially) checking the authorization multiple times:

```ruby
def requested_resource
  @_requested_resource ||= find_resource(params[:id]).tap do |r|
    authorize_resource(r)
  end
end
```

But then I think... we could go further. If the spec for `authorize_resource` required it to return the resource on success, we could do this:

```ruby
def requested_resource
  @_requested_resource ||= authorize_resource(find_resource(params[:id]))
end
```

In fact that's what Pundit's own `authorize` does, which in turn means that the implementation provided by `Punditize` does this, although not the default, blank implementation.

But I'm probably complicating this too much for too little gain, so I'll be happy with something along the lines of my first alternative.